src/views/ModelPlan/TaskList/_components/TaskListStatus/index.tsx

```diff
@@ -41,7 +43,11 @@ const TaskListStatus = ({
 
       <div>
         <UswdsReactLink
-          to={`/models/${modelID}/status`}
+          to={
+            readOnly
+              ? `/models/${modelID}/status#read-only`
+              : `/models/${modelID}/status`
+          }
```

Comment:
I considered passing a state to the location, but honestly I think adding a hash to the url works just as well!  Kinda annoying that location state doesn't allow for boolean prop ... 

```
to={{
            pathname: `/models/${modelID}/status`,
            state: { readOnly: readOnly ? 'true' : null }
          }}
```